### ISSUE TITLE
Add aria-current to current page

### ISSUE THREAD

**Issue by MatzeKitt** [lloc/Multisite-Language-Switcher#458]
**Describe the bug**
If the current language is displayed, the `aria-current` attribute is missing and thus not indicating the user that it’s the link to the current page.

**To Reproduce**
1. Enable displaying current language in the settings.
2. Using language switcher via screen reader in the frontend.

**Expected behaviour**
If the current language is displayed, add `aria-current="page"` to the link, indicating it’s the current page.
